meson.build

```diff
@@ -0,0 +1,269 @@
+project('WPEBackend-fdo', ['c', 'cpp'],
+	meson_version: '>=0.49',
+	default_options: [
+		'b_ndebug=if-release',
+		'c_std=c99',
+		'cpp_eh=none',
+		'cpp_std=c++11',
+	],
+	license: 'BSD-2-Clause',
+	version: '1.7.0',
+)
+
+# This refers to the API level provided. This is modified only with major,
+# breaking changes, which need modifications in programs using the library
+# before they can be compiled again.
+api_version = '1.0'
+
+# Before making a release, the LT_VERSION string should be modified.
```

Comment:
But there is no LT_VERSION, is there? :) Replace the comment with this one:

https://gitlab.gnome.org/GNOME/gnome-desktop/-/blob/1eb198b9c5fc1e92cf5aa83cae2ab1a50762358f/meson.build#L7

Reply:
Doing as suggested results in a different installed library name, and a different shared object version as well. We need to make the build with Meson generate exactly the same as for CMake, so for now we don't have any other option but keep calculating the `soversion` in the same way as done for the CMake build :shrug:

Reply:
Just manually set it to the exact same thing produced by the CMake build. Currently that's libWPEBackend-fdo-1.0.so.1.6.1, so you can start the version number at 1.6.1, and go from there. Much easier than CRA.

If you *really* want to use CRA, then at least remove the references to libtool. That wasn't even used by CMake. :P

Reply:
Then we would have different numbers in `CMakeLists.txt` and `meson.build`, and I would rather keep them the same. Let's keep using *(C, R, A)* for now, and we can replace it later on when we remove the CMake build system in about a year :wink: